app/views/core/SubscribeModal.coffee

```diff
@@ -114,10 +148,13 @@ module.exports = class SubscribeModal extends ModalView
       jqxhr = me.subscribe(token)
       return Promise.resolve(jqxhr)
     .then =>
-      application.tracker?.trackEvent 'Finished subscription purchase', value: @purchasedAmount
-      @onSubscriptionSuccess() 
+      application.tracker?.trackEvent 'Finished subscription purchase', { value: @purchasedAmount, service: 'stripe' }
+      @onSubscriptionSuccess()
     .catch (jqxhr) =>
       return unless jqxhr # in case of cancellations
+      stripe = me.get('stripe') ? {}
+      delete stripe.token
+      delete stripe.planID
```

Comment:
So I do need them here, but not in the other spot, right? `@selectedProduct.purchase` is the new one?